src/foam/core/types.js

```diff
@@ -890,6 +890,33 @@ foam.CLASS({
         return foam.String.daoize(of.name);
       }
     },
+    {
+      class: 'Boolean',
+      name: 'enableLink',
+      documentation: `
+        Create the reference view as an anchor link to the reference's DetailView or provided menu.
+        Check ReadReferenceView documentation for more info.
+      `,
+      value: true
+    },
+    {
+      class: 'Boolean',
+      name: 'controlAccessToDAOSummary',
```

Comment:
don't think this is needed. The auth check handles access to dao summary - so what reason is there for this?

Reply:
Only when this is set, then we will do auth check. There might be some classes that you don't want to do auth check on. Without this property, you lose that ability.

Reply:
Only when this is set, we will do auth check. There might be some classes or even some cases that you don't want to do auth check. Having this property gives you flexibility to only use permission where needed.

Reply:
Only when this is set, we will do auth check. There might be some classes or even some cases that you don't want to do auth check. Having this property gives you flexibility to only use permission where needed.

Reply:
I think that if we want to control access, then we should do it through the menus. Setting up a DAO2 menu is easy and already gives us the appropriate permission support.